basics/node.md

```diff
@@ -0,0 +1,26 @@
+# Interoperating with Node and NPM
+
+While Deno is pretty powerful itself, many people will want to leverage code and
+libraries that are built for [Node](https://nodejs.org/), in particular the
+large set of packages available on the [NPM](https://npmjs.com/) registry.
```

Comment:
always _npm_ according to their brand guidelines: https://docs.npmjs.com/policies/logos-and-usage